INFO 2024-04-27 15:17:34,947 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:21:22,381 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:24:29,086 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:24:29,506 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:25:18,257 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 15:33:36,008 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 15:33:36,010 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2116
INFO 2024-04-27 15:33:47,514 basehttp 212 "GET /admin/ HTTP/1.1" 302 0
INFO 2024-04-27 15:33:47,556 basehttp 212 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4172
INFO 2024-04-27 15:33:47,741 basehttp 212 "GET /static/admin/css/base.css HTTP/1.1" 200 21544
INFO 2024-04-27 15:33:47,744 basehttp 212 "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2682
INFO 2024-04-27 15:33:47,746 basehttp 212 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
INFO 2024-04-27 15:33:47,748 basehttp 212 "GET /static/admin/css/login.css HTTP/1.1" 200 958
INFO 2024-04-27 15:33:47,750 basehttp 212 "GET /static/admin/css/responsive.css HTTP/1.1" 200 17905
INFO 2024-04-27 15:33:47,753 basehttp 212 "GET /static/admin/js/theme.js HTTP/1.1" 200 1943
INFO 2024-04-27 15:33:47,756 basehttp 212 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
WARNING 2024-04-27 15:33:59,404 log 241 Not Found: /image_codes/c38ee8f3-1857-4ec4-909e-c29d8ddfd31d/
WARNING 2024-04-27 15:33:59,404 basehttp 212 "GET /image_codes/c38ee8f3-1857-4ec4-909e-c29d8ddfd31d/ HTTP/1.1" 404 2224
WARNING 2024-04-27 15:34:08,090 log 241 Not Found: /mobiles/17369236240/count/
WARNING 2024-04-27 15:34:08,091 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 404 2155
WARNING 2024-04-27 15:34:08,314 log 241 Not Found: /mobiles/17369236240/count/
WARNING 2024-04-27 15:34:08,314 log 241 Not Found: /sms_codes/17369236240/
WARNING 2024-04-27 15:34:08,315 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 404 2155
WARNING 2024-04-27 15:34:08,315 basehttp 212 "GET /sms_codes/17369236240/?image_code=&image_code_id=c38ee8f3-1857-4ec4-909e-c29d8ddfd31d HTTP/1.1" 404 2210
INFO 2024-04-27 15:37:34,034 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:37:34,413 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:39:34,528 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:39:35,062 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:42:04,483 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:42:04,969 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:43:17,242 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:43:17,716 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:43:33,727 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:43:34,216 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:43:37,630 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:43:38,120 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:44:11,779 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:44:12,214 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:44:49,795 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:44:50,288 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:45:46,875 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:45:47,269 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:46:06,050 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:46:06,473 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:46:55,568 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:46:55,966 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:47:35,628 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:47:36,027 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:47:51,722 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:47:52,126 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:48:09,036 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 15:48:09,440 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 15:49:00,486 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 15:49:00,914 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:19:21,257 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:20:13,309 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:20:32,085 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 16:20:32,545 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:20:37,241 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:20:37,241 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:20:37,249 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:20:37,250 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:20:37,257 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:20:37,257 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:20:51,531 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:20:51,531 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:20:51,536 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:20:51,536 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:20:51,541 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:20:51,541 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
INFO 2024-04-27 16:21:01,897 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:21:04,240 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:04,241 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:04,248 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:04,249 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:04,367 log 241 Not Found: /favicon.ico
WARNING 2024-04-27 16:21:04,367 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2290
WARNING 2024-04-27 16:21:18,890 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:18,890 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:18,895 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:18,895 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:18,899 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:18,900 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:18,905 log 241 Not Found: /favicon.ico
WARNING 2024-04-27 16:21:18,906 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2290
INFO 2024-04-27 16:21:27,877 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:21:30,022 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:30,023 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:30,029 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:30,030 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:30,113 log 241 Not Found: /favicon.ico
WARNING 2024-04-27 16:21:30,114 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2290
WARNING 2024-04-27 16:21:34,678 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:34,678 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:34,684 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:34,685 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:34,787 log 241 Not Found: /favicon.ico
WARNING 2024-04-27 16:21:34,788 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2290
WARNING 2024-04-27 16:21:38,377 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:38,379 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
INFO 2024-04-27 16:21:46,402 basehttp 212 "GET /admin/ HTTP/1.1" 302 0
INFO 2024-04-27 16:21:46,430 basehttp 212 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4172
WARNING 2024-04-27 16:21:59,877 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:59,878 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:59,883 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:21:59,883 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:21:59,961 log 241 Not Found: /favicon.ico
WARNING 2024-04-27 16:21:59,961 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2290
INFO 2024-04-27 16:22:36,062 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:22:38,617 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:22:38,617 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:22:38,628 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:22:38,628 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:22:38,634 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:22:38,635 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2296
WARNING 2024-04-27 16:22:39,762 log 241 Not Found: /favicon.ico
WARNING 2024-04-27 16:22:39,762 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2290
WARNING 2024-04-27 16:23:06,180 log 241 Not Found: /username/admin/count
WARNING 2024-04-27 16:23:06,181 basehttp 212 "GET /username/admin/count HTTP/1.1" 404 2317
ERROR 2024-04-27 16:24:25,474 basehttp 192 You're accessing the development server over HTTPS, but it only supports HTTP.
WARNING 2024-04-27 16:24:44,119 log 241 Not Found: /username/admin/count
WARNING 2024-04-27 16:24:44,119 basehttp 212 "POST /username/admin/count HTTP/1.1" 404 2318
ERROR 2024-04-27 16:24:59,655 log 241 Internal Server Error: /usernames/admin/count
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\middleware\common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\middleware\common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to localhost:8000/usernames/admin/count/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
ERROR 2024-04-27 16:24:59,657 basehttp 212 "POST /usernames/admin/count HTTP/1.1" 500 76457
ERROR 2024-04-27 16:26:02,414 log 241 Internal Server Error: /usernames/admin/count
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\middleware\common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\middleware\common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to localhost:8000/usernames/admin/count/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
ERROR 2024-04-27 16:26:02,415 basehttp 212 "POST /usernames/admin/count HTTP/1.1" 500 76457
INFO 2024-04-27 16:26:27,780 basehttp 212 "GET /usernames/admin/count HTTP/1.1" 301 0
INFO 2024-04-27 16:26:27,788 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 16:30:17,327 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\urls.py changed, reloading.
INFO 2024-04-27 16:30:17,805 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:30:42,755 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\urls.py changed, reloading.
INFO 2024-04-27 16:30:43,244 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:30:53,094 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\urls.py changed, reloading.
INFO 2024-04-27 16:30:53,590 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:31:12,814 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\urls.py changed, reloading.
INFO 2024-04-27 16:31:13,224 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:35:48,431 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:35:51,378 log 241 Not Found: /usernames/admin/count
WARNING 2024-04-27 16:35:51,378 basehttp 212 "GET /usernames/admin/count HTTP/1.1" 404 2329
INFO 2024-04-27 16:36:09,919 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\utils\convertors.py changed, reloading.
INFO 2024-04-27 16:36:10,439 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:36:12,770 log 241 Not Found: /usernames/admin/count
WARNING 2024-04-27 16:36:12,770 basehttp 212 "GET /usernames/admin/count HTTP/1.1" 404 2329
INFO 2024-04-27 16:36:21,572 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:36:24,108 log 241 Not Found: /usernames/admin/count
WARNING 2024-04-27 16:36:24,110 basehttp 212 "GET /usernames/admin/count HTTP/1.1" 404 2329
INFO 2024-04-27 16:36:39,955 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\utils\convertors.py changed, reloading.
INFO 2024-04-27 16:36:40,384 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:36:43,188 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:36:45,876 log 241 Not Found: /usernames/admin/count
WARNING 2024-04-27 16:36:45,877 basehttp 212 "GET /usernames/admin/count HTTP/1.1" 404 2329
INFO 2024-04-27 16:37:20,300 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\utils\convertors.py changed, reloading.
INFO 2024-04-27 16:37:20,770 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:37:23,146 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:37:25,316 basehttp 212 "GET /usernames/admin/count HTTP/1.1" 301 0
INFO 2024-04-27 16:37:25,324 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
WARNING 2024-04-27 16:39:25,715 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:39:25,716 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2305
WARNING 2024-04-27 16:39:26,544 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:39:26,544 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2305
WARNING 2024-04-27 16:39:30,882 log 241 Not Found: /carts/simple/
WARNING 2024-04-27 16:39:30,884 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2305
WARNING 2024-04-27 16:39:36,854 log 241 Not Found: /image_codes/470f764f-189d-451d-adfa-3199e2f6dd71/
WARNING 2024-04-27 16:39:36,854 basehttp 212 "GET /image_codes/470f764f-189d-451d-adfa-3199e2f6dd71/ HTTP/1.1" 404 2413
INFO 2024-04-27 16:39:43,195 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
WARNING 2024-04-27 16:39:46,008 log 241 Not Found: /image_codes/fb34bb22-d921-4e36-b3f6-c441f3e83482/
WARNING 2024-04-27 16:39:46,008 basehttp 212 "GET /image_codes/fb34bb22-d921-4e36-b3f6-c441f3e83482/ HTTP/1.1" 404 2413
INFO 2024-04-27 16:39:53,013 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 16:39:54,751 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 16:42:49,566 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:42:50,031 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:43:12,281 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:43:12,646 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:43:42,733 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:43:43,123 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:44:09,069 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:44:09,496 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:44:12,043 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:46:33,116 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:47:18,347 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:47:18,817 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:47:33,881 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:47:34,392 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-27 16:47:52,039 log 241 Not Found: /image_codes/05d8d857-d2dc-44f0-b486-80b3c3d7d02a/
WARNING 2024-04-27 16:47:52,039 basehttp 212 "GET /image_codes/05d8d857-d2dc-44f0-b486-80b3c3d7d02a/ HTTP/1.1" 404 2413
INFO 2024-04-27 16:48:48,137 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:48:48,570 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:49:00,481 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:49:00,985 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:49:26,425 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:49:26,942 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:49:38,901 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:49:39,366 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:49:58,588 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:49:59,039 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:50:20,367 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:50:20,852 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:50:34,814 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 16:50:35,304 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 16:50:52,650 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 16:52:09,732 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\models.py changed, reloading.
INFO 2024-04-27 16:52:10,215 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:32:49,001 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:33:32,044 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:34:00,742 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:34:03,375 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
WARNING 2024-04-27 23:34:04,455 log 241 Not Found: /image_codes/b21f0061-7ea5-48fa-b5b2-8aaeaecf9500/
WARNING 2024-04-27 23:34:04,456 basehttp 212 "GET /image_codes/b21f0061-7ea5-48fa-b5b2-8aaeaecf9500/ HTTP/1.1" 404 2596
INFO 2024-04-27 23:34:09,747 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:34:11,124 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:34:17,288 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:34:18,783 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:35:12,917 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:35:13,416 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:37:51,930 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:43:44,705 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:43:45,239 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:43:49,923 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:43:50,424 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:44:25,315 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:44:25,712 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:45:05,705 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:45:06,206 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:45:20,315 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:45:20,805 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:45:23,398 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:45:23,905 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:46:13,457 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:46:13,946 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:47:37,076 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:47:37,579 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:48:01,490 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:48:01,925 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:48:55,960 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:48:56,413 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:49:02,821 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:49:03,244 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:49:06,525 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:49:06,980 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:50:10,001 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:50:10,435 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:51:52,908 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\urls.py changed, reloading.
INFO 2024-04-27 23:51:53,397 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:52:53,389 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-27 23:52:53,921 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:53:05,883 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:53:21,213 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:53:21,215 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:53:21,216 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
ERROR 2024-04-27 23:53:21,457 log 241 Internal Server Error: /register/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py", line 83, in post
    login(request, user)
    ^^^^^
NameError: name 'login' is not defined
ERROR 2024-04-27 23:53:21,459 basehttp 212 "POST /register/ HTTP/1.1" 500 83661
INFO 2024-04-27 23:53:22,738 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:53:22,740 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:53:22,740 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:53:22,956 basehttp 212 "POST /register/ HTTP/1.1" 200 51
INFO 2024-04-27 23:53:32,367 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:53:32,369 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:53:32,369 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:53:32,586 basehttp 212 "POST /register/ HTTP/1.1" 200 51
INFO 2024-04-27 23:54:16,115 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:54:16,688 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:54:24,541 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:54:24,551 basehttp 212 "GET /usernames/admin/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:24,551 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:24,765 basehttp 212 "POST /register/ HTTP/1.1" 200 51
INFO 2024-04-27 23:54:30,588 basehttp 212 "GET /usernames/admins/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:30,800 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:54:30,802 basehttp 212 "GET /usernames/admins/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:30,803 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:31,020 basehttp 212 "POST /register/ HTTP/1.1" 200 51
INFO 2024-04-27 23:54:41,023 basehttp 212 "GET /mobiles/17369336230/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:41,245 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:54:41,246 basehttp 212 "GET /mobiles/17369336230/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:41,247 basehttp 212 "GET /usernames/admins/count/ HTTP/1.1" 200 39
ERROR 2024-04-27 23:54:41,483 log 241 Internal Server Error: /register/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 332, in check_response
    raise ValueError(
ValueError: The view apps.users.views.view didn't return an HttpResponse object. It returned None instead.
ERROR 2024-04-27 23:54:41,483 basehttp 212 "POST /register/ HTTP/1.1" 500 73912
INFO 2024-04-27 23:54:45,483 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:54:45,486 basehttp 212 "GET /usernames/admins/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:45,486 basehttp 212 "GET /mobiles/17369336230/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:54:45,710 basehttp 212 "POST /register/ HTTP/1.1" 200 51
INFO 2024-04-27 23:55:52,271 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:55:52,732 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:58:29,067 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:58:29,575 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:59:07,658 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-27 23:59:08,155 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-27 23:59:13,591 basehttp 212 "GET /mobiles/17369336231/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:59:13,806 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:59:13,807 basehttp 212 "GET /usernames/admins/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:59:13,808 basehttp 212 "GET /mobiles/17369336231/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:59:14,017 basehttp 212 "POST /register/ HTTP/1.1" 200 51
INFO 2024-04-27 23:59:18,395 basehttp 212 "GET /usernames/admin5/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:59:18,605 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-27 23:59:18,608 basehttp 212 "GET /usernames/admin5/count/ HTTP/1.1" 200 39
INFO 2024-04-27 23:59:18,609 basehttp 212 "GET /mobiles/17369336231/count/ HTTP/1.1" 200 39
ERROR 2024-04-27 23:59:18,833 log 241 Internal Server Error: /register/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py", line 83, in post
    login(request, user)
    ^^^^^
NameError: name 'login' is not defined
ERROR 2024-04-27 23:59:18,835 basehttp 212 "POST /register/ HTTP/1.1" 500 83667
INFO 2024-04-28 00:00:33,739 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 00:00:34,242 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:00:41,772 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 00:00:42,277 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:00:50,864 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 00:00:51,364 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:01:00,215 basehttp 212 "GET /mobiles/17369336234/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:01:02,153 basehttp 212 "GET /usernames/admin7/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:01:02,367 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-28 00:01:02,368 basehttp 212 "GET /usernames/admin7/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:01:02,368 basehttp 212 "GET /mobiles/17369336234/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:01:02,595 basehttp 212 "POST /register/ HTTP/1.1" 200 27
WARNING 2024-04-28 00:01:03,895 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:01:03,895 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2644
INFO 2024-04-28 00:02:08,930 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 00:02:09,392 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:10:35,696 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 00:10:36,240 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:11:47,690 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-28 00:11:48,184 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:12:39,016 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-28 00:12:39,510 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:20:27,280 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\settings.py changed, reloading.
INFO 2024-04-28 00:20:27,915 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:26:00,017 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\tuling\urls.py changed, reloading.
INFO 2024-04-28 00:26:00,456 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:31:21,292 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:34:24,876 autoreload 668 Watching for file changes with StatReloader
ERROR 2024-04-28 00:34:30,586 log 241 Internal Server Error: /image_codes/3f1bd7b8-7e20-4322-b6c3-81aeeece90d2/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:30,589 basehttp 212 "GET /image_codes/3f1bd7b8-7e20-4322-b6c3-81aeeece90d2/ HTTP/1.1" 500 97687
ERROR 2024-04-28 00:34:32,312 log 241 Internal Server Error: /image_codes/52a20982-2712-4774-8629-8d73d7ca767b/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:32,313 basehttp 212 "GET /image_codes/52a20982-2712-4774-8629-8d73d7ca767b/ HTTP/1.1" 500 97686
ERROR 2024-04-28 00:34:33,007 log 241 Internal Server Error: /image_codes/5c7575f9-8c46-435c-991f-9628d1dc97e0/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:33,010 basehttp 212 "GET /image_codes/5c7575f9-8c46-435c-991f-9628d1dc97e0/ HTTP/1.1" 500 97686
ERROR 2024-04-28 00:34:33,692 log 241 Internal Server Error: /image_codes/25df560f-67d1-4fab-a423-495419d30c55/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:33,693 basehttp 212 "GET /image_codes/25df560f-67d1-4fab-a423-495419d30c55/ HTTP/1.1" 500 97688
ERROR 2024-04-28 00:34:34,441 log 241 Internal Server Error: /image_codes/f0a6b384-b722-42df-b398-fba2e9abb902/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:34,443 basehttp 212 "GET /image_codes/f0a6b384-b722-42df-b398-fba2e9abb902/ HTTP/1.1" 500 97687
ERROR 2024-04-28 00:34:35,764 log 241 Internal Server Error: /image_codes/f2e16cfb-b38b-4772-b7a7-d3ad155970f0/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:35,766 basehttp 212 "GET /image_codes/f2e16cfb-b38b-4772-b7a7-d3ad155970f0/ HTTP/1.1" 500 97688
ERROR 2024-04-28 00:34:36,908 log 241 Internal Server Error: /image_codes/9a7900e5-070a-43bc-834c-f386f746c92b/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:36,909 basehttp 212 "GET /image_codes/9a7900e5-070a-43bc-834c-f386f746c92b/ HTTP/1.1" 500 97687
ERROR 2024-04-28 00:34:37,521 log 241 Internal Server Error: /image_codes/22c06cfc-6a9b-4694-874a-7cb61037f145/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 12, in get
    text,image=captcha.captcha.generate_captcha()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 215, in generate_captcha
    return self.captcha("")
           ^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 204, in captcha
    image = self.text(image, self.fonts, drawings=['warp', 'rotate', 'offset'])
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\libs\captcha\captcha\captcha.py", line 132, in text
    c_width, c_height = draw.textsize(c, font=font)
                        ^^^^^^^^^^^^^
AttributeError: 'ImageDraw' object has no attribute 'textsize'
ERROR 2024-04-28 00:34:37,523 basehttp 212 "GET /image_codes/22c06cfc-6a9b-4694-874a-7cb61037f145/ HTTP/1.1" 500 97687
INFO 2024-04-28 00:37:21,796 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:39:32,456 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:40:40,770 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:40:44,669 basehttp 212 "GET /image_codes/0d1b2f1a-b556-436a-b1a3-ebce84079346/ HTTP/1.1" 200 3338
INFO 2024-04-28 00:43:34,000 basehttp 212 "GET /image_codes/c906e667-701c-4eea-a98c-e85b520fc040/ HTTP/1.1" 200 3253
INFO 2024-04-28 00:43:34,539 basehttp 212 "GET /image_codes/29bb6e5a-5e7e-4747-9210-cc98e1d41d7b/ HTTP/1.1" 200 3515
INFO 2024-04-28 00:43:35,232 basehttp 212 "GET /image_codes/6fe78202-0d30-491e-a858-6a3c0bf79b22/ HTTP/1.1" 200 3390
INFO 2024-04-28 00:44:45,520 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\utils\convertors.py changed, reloading.
INFO 2024-04-28 00:44:46,047 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:45:10,606 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\utils\convertors.py changed, reloading.
INFO 2024-04-28 00:45:11,104 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:46:02,888 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 00:46:03,315 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:46:17,486 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 00:46:18,102 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:46:33,309 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 00:46:33,837 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:46:52,156 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 00:46:52,784 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 00:46:58,559 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 00:46:59,069 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 00:47:02,912 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:47:02,913 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
WARNING 2024-04-28 00:47:02,921 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:47:02,923 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
ERROR 2024-04-28 00:47:14,640 log 241 Internal Server Error: /image_codes/33526481-a7f7-4b2d-95ea-4910da88d54d/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 14, in get
    redis_cil.setex(uuid,100,text)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 2367, in setex
    return self.execute_command("SETEX", name, time, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 521, in _send_command_parse_response
    conn.send_command(*args)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 477, in send_command
    self._command_packer.pack(*args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 96, in pack
    for arg in map(self.encode, args):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\_parsers\encoders.py", line 29, in encode
    raise DataError(
redis.exceptions.DataError: Invalid input of type: 'UUID'. Convert to a bytes, string, int or float first.
ERROR 2024-04-28 00:47:14,644 basehttp 212 "GET /image_codes/33526481-a7f7-4b2d-95ea-4910da88d54d/ HTTP/1.1" 500 124084
ERROR 2024-04-28 00:47:15,770 log 241 Internal Server Error: /image_codes/6df77686-0bd7-461e-8b95-c11c05c29716/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 14, in get
    redis_cil.setex(uuid,100,text)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 2367, in setex
    return self.execute_command("SETEX", name, time, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 521, in _send_command_parse_response
    conn.send_command(*args)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 477, in send_command
    self._command_packer.pack(*args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 96, in pack
    for arg in map(self.encode, args):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\_parsers\encoders.py", line 29, in encode
    raise DataError(
redis.exceptions.DataError: Invalid input of type: 'UUID'. Convert to a bytes, string, int or float first.
ERROR 2024-04-28 00:47:15,773 basehttp 212 "GET /image_codes/6df77686-0bd7-461e-8b95-c11c05c29716/ HTTP/1.1" 500 124102
ERROR 2024-04-28 00:47:20,622 log 241 Internal Server Error: /image_codes/c98a64cc-f82f-46c8-8a03-2354114b06de/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 14, in get
    redis_cil.setex(uuid,100,text)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 2367, in setex
    return self.execute_command("SETEX", name, time, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 521, in _send_command_parse_response
    conn.send_command(*args)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 477, in send_command
    self._command_packer.pack(*args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 96, in pack
    for arg in map(self.encode, args):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\_parsers\encoders.py", line 29, in encode
    raise DataError(
redis.exceptions.DataError: Invalid input of type: 'UUID'. Convert to a bytes, string, int or float first.
ERROR 2024-04-28 00:47:20,624 basehttp 212 "GET /image_codes/c98a64cc-f82f-46c8-8a03-2354114b06de/ HTTP/1.1" 500 124087
ERROR 2024-04-28 00:47:24,384 log 241 Internal Server Error: /image_codes/cead2d67-6f2e-46d6-a826-1660031b94e3/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 14, in get
    redis_cil.setex(uuid,100,text)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 2367, in setex
    return self.execute_command("SETEX", name, time, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 521, in _send_command_parse_response
    conn.send_command(*args)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 477, in send_command
    self._command_packer.pack(*args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 96, in pack
    for arg in map(self.encode, args):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\_parsers\encoders.py", line 29, in encode
    raise DataError(
redis.exceptions.DataError: Invalid input of type: 'UUID'. Convert to a bytes, string, int or float first.
ERROR 2024-04-28 00:47:24,387 basehttp 212 "GET /image_codes/cead2d67-6f2e-46d6-a826-1660031b94e3/ HTTP/1.1" 500 124077
ERROR 2024-04-28 00:47:25,106 log 241 Internal Server Error: /image_codes/d4eda1f8-8dc5-4e20-91d8-324d53d395da/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 14, in get
    redis_cil.setex(uuid,100,text)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 2367, in setex
    return self.execute_command("SETEX", name, time, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 521, in _send_command_parse_response
    conn.send_command(*args)
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 477, in send_command
    self._command_packer.pack(*args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 96, in pack
    for arg in map(self.encode, args):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\_parsers\encoders.py", line 29, in encode
    raise DataError(
redis.exceptions.DataError: Invalid input of type: 'UUID'. Convert to a bytes, string, int or float first.
ERROR 2024-04-28 00:47:25,109 basehttp 212 "GET /image_codes/d4eda1f8-8dc5-4e20-91d8-324d53d395da/ HTTP/1.1" 500 124083
INFO 2024-04-28 00:47:42,165 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 00:47:42,714 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 00:47:46,003 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:47:46,005 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
WARNING 2024-04-28 00:47:49,281 log 241 Not Found: /register.html
WARNING 2024-04-28 00:47:49,282 basehttp 212 "GET /register.html HTTP/1.1" 404 2821
WARNING 2024-04-28 00:47:49,412 log 241 Not Found: /favicon.ico
WARNING 2024-04-28 00:47:49,414 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2815
WARNING 2024-04-28 00:47:50,442 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:47:50,443 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
INFO 2024-04-28 00:48:05,835 basehttp 212 "GET /admin/ HTTP/1.1" 302 0
INFO 2024-04-28 00:48:05,865 basehttp 212 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4172
INFO 2024-04-28 00:48:05,921 basehttp 212 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2024-04-28 00:48:05,922 basehttp 212 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2024-04-28 00:48:05,924 basehttp 212 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2024-04-28 00:48:05,924 basehttp 212 "GET /static/admin/css/login.css HTTP/1.1" 304 0
INFO 2024-04-28 00:48:05,926 basehttp 212 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2024-04-28 00:48:05,941 basehttp 212 "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2024-04-28 00:48:05,941 basehttp 212 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
WARNING 2024-04-28 00:48:06,006 log 241 Not Found: /favicon.ico
WARNING 2024-04-28 00:48:06,007 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2815
WARNING 2024-04-28 00:48:15,337 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:48:15,337 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
INFO 2024-04-28 00:48:18,008 basehttp 212 "GET /image_codes/7d4ead1a-84bd-458d-9da5-3fdeacc0ee0e/ HTTP/1.1" 200 3009
INFO 2024-04-28 00:48:19,362 basehttp 212 "GET /image_codes/69601566-e9b0-49ed-a21b-a099615a01b2/ HTTP/1.1" 200 3536
WARNING 2024-04-28 00:50:11,309 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:50:11,309 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
INFO 2024-04-28 00:50:17,769 basehttp 212 "GET /image_codes/51036015-1e8a-448e-8c12-cdc66d476d84/ HTTP/1.1" 200 2723
INFO 2024-04-28 00:50:19,209 basehttp 212 "GET /image_codes/92d45367-8ac0-40f2-a781-f6292743f189/ HTTP/1.1" 200 3367
INFO 2024-04-28 00:50:39,756 basehttp 212 "GET /usernames/yanminghe/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:50:51,683 basehttp 212 "GET /mobiles/17369236245/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:51:05,465 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-28 00:51:05,469 basehttp 212 "GET /usernames/yanminghe/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:51:05,470 basehttp 212 "GET /mobiles/17369236245/count/ HTTP/1.1" 200 39
INFO 2024-04-28 00:51:05,672 basehttp 212 "POST /register/ HTTP/1.1" 200 27
WARNING 2024-04-28 00:51:05,806 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 00:51:05,807 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 2821
INFO 2024-04-28 09:29:10,075 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:29:30,657 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:29:31,097 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:29:49,002 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:29:49,755 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:30:11,994 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:30:12,506 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:30:30,384 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:30:31,074 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:30:41,814 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:30:42,320 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:30:59,305 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:30:59,936 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:31:04,816 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:31:05,400 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:31:18,500 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:31:18,970 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:31:47,866 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:31:48,341 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:31:53,712 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:31:54,218 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:31:59,694 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:32:00,313 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:32:06,831 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:32:07,349 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:32:29,938 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:32:30,472 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:32:37,342 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:32:37,857 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:32:46,882 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:32:47,288 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:33:05,171 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:33:05,794 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:33:17,527 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:33:17,920 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:33:21,229 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:33:21,710 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:33:29,634 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:33:30,210 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:33:33,558 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\urls.py changed, reloading.
INFO 2024-04-28 09:33:34,197 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 09:33:58,661 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 09:33:58,662 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 09:34:03,886 basehttp 212 "GET /image_codes/7d011d00-b98e-44e3-9852-3bd35f65b741/ HTTP/1.1" 200 3072
INFO 2024-04-28 09:34:18,185 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:36:00,938 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:00,961 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:00,963 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:02,789 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:02,806 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:02,808 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:03,573 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:03,590 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:03,592 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:03,817 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:03,833 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:03,835 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:05,424 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:05,438 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:05,441 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:05,683 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:05,696 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:05,698 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:05,905 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:05,920 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:05,922 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:07,659 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:07,674 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:07,676 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:09,448 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:09,462 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:09,464 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:09,597 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:09,608 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:09,609 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:36:09,812 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:36:09,824 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 23, in get
    if not all(image_code,uuid):
           ^^^^^^^^^^^^^^^^^^^^
TypeError: all() takes exactly one argument (2 given)
ERROR 2024-04-28 09:36:09,825 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 500 85425
INFO 2024-04-28 09:38:35,847 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:38:36,455 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:38:40,751 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:38:40,756 basehttp 212 "GET /sms_codes/17752100458/?image_code=EMCB&image_code_id=7d011d00-b98e-44e3-9852-3bd35f65b741 HTTP/1.1" 200 69
INFO 2024-04-28 09:39:35,986 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:39:40,354 basehttp 212 "GET /image_codes/1b23556a-d794-458f-9472-e7b514f83a71/ HTTP/1.1" 200 3147
INFO 2024-04-28 09:39:51,517 basehttp 212 "GET /sms_codes/17752100458/?image_code=\TKX6&image_code_id=1b23556a-d794-458f-9472-e7b514f83a71 HTTP/1.1" 200 69
INFO 2024-04-28 09:39:51,520 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
WARNING 2024-04-28 09:40:26,787 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 09:40:26,788 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
WARNING 2024-04-28 09:40:27,593 log 241 Not Found: /favicon.ico
WARNING 2024-04-28 09:40:27,594 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2994
INFO 2024-04-28 09:41:32,605 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 09:41:39,811 log 241 Not Found: /register.html
WARNING 2024-04-28 09:41:39,812 basehttp 212 "GET /register.html HTTP/1.1" 404 3000
WARNING 2024-04-28 09:41:49,495 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 09:41:49,497 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 09:41:52,095 basehttp 212 "GET /image_codes/1391a659-6d7e-4776-b5f0-be5c3a05c140/ HTTP/1.1" 200 2868
INFO 2024-04-28 09:42:01,078 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:42:01,295 basehttp 212 "GET /image_codes/d87cfcd6-d16a-4e59-b552-fbe02bc93597/ HTTP/1.1" 200 3161
INFO 2024-04-28 09:42:02,548 basehttp 212 "GET /image_codes/d1dcca67-389c-4d20-bcd2-3787aec51775/ HTTP/1.1" 200 3065
INFO 2024-04-28 09:42:12,001 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:43:31,915 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:43:33,344 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:43:47,514 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:44:05,808 basehttp 212 "GET /image_codes/bd834257-4e54-4081-9619-be1d66bc34dc/ HTTP/1.1" 200 3000
INFO 2024-04-28 09:44:25,739 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:44:43,634 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:44:54,619 basehttp 212 "GET /image_codes/86f1f9f8-4b94-441c-8435-2530ca069d7a/ HTTP/1.1" 200 3242
INFO 2024-04-28 09:45:01,739 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:45:09,566 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:45:30,445 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 33, in get
    sms_code = '%06d' % random.randint(0,999999)
                        ^^^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'randint'
ERROR 2024-04-28 09:45:41,575 basehttp 212 "GET /sms_codes/17752100458/?image_code=JEWQ&image_code_id=86f1f9f8-4b94-441c-8435-2530ca069d7a HTTP/1.1" 500 99256
INFO 2024-04-28 09:46:08,198 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:46:11,745 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:46:11,769 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 33, in get
    sms_code = '%06d' % random.randint(0,999999)
                        ^^^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'randint'
ERROR 2024-04-28 09:46:11,770 basehttp 212 "GET /sms_codes/17752100458/?image_code=JEWQ&image_code_id=86f1f9f8-4b94-441c-8435-2530ca069d7a HTTP/1.1" 500 85324
INFO 2024-04-28 09:46:13,307 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:46:13,314 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 33, in get
    sms_code = '%06d' % random.randint(0,999999)
                        ^^^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'randint'
ERROR 2024-04-28 09:46:13,315 basehttp 212 "GET /sms_codes/17752100458/?image_code=JEWQ&image_code_id=86f1f9f8-4b94-441c-8435-2530ca069d7a HTTP/1.1" 500 85324
INFO 2024-04-28 09:46:15,085 basehttp 212 "GET /image_codes/380e3d4e-458d-42c3-9a40-ae2237bc4bb6/ HTTP/1.1" 200 3358
INFO 2024-04-28 09:46:21,285 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:46:25,865 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:46:25,874 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 33, in get
    sms_code = '%06d' % random.randint(0,999999)
                        ^^^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'randint'
ERROR 2024-04-28 09:46:25,875 basehttp 212 "GET /sms_codes/17752100458/?image_code=FBMF&image_code_id=380e3d4e-458d-42c3-9a40-ae2237bc4bb6 HTTP/1.1" 500 85324
INFO 2024-04-28 09:46:28,034 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:46:28,045 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 33, in get
    sms_code = '%06d' % random.randint(0,999999)
                        ^^^^^^^^^^^^^^
AttributeError: 'builtin_function_or_method' object has no attribute 'randint'
ERROR 2024-04-28 09:46:28,046 basehttp 212 "GET /sms_codes/17752100458/?image_code=FBMF&image_code_id=380e3d4e-458d-42c3-9a40-ae2237bc4bb6 HTTP/1.1" 500 85324
INFO 2024-04-28 09:47:16,774 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:47:17,184 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:47:21,060 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 09:47:21,085 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 33, in get
    sms_code = '%06d' % randint(0,999999)
                        ^^^^^^^
NameError: name 'randint' is not defined
ERROR 2024-04-28 09:47:21,088 basehttp 212 "GET /sms_codes/17752100458/?image_code=FBMF&image_code_id=380e3d4e-458d-42c3-9a40-ae2237bc4bb6 HTTP/1.1" 500 85094
INFO 2024-04-28 09:48:25,407 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 09:48:26,059 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:48:30,194 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:48:30,198 basehttp 212 "GET /sms_codes/17752100458/?image_code=FBMF&image_code_id=380e3d4e-458d-42c3-9a40-ae2237bc4bb6 HTTP/1.1" 200 69
INFO 2024-04-28 09:49:19,672 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:49:22,088 basehttp 212 "GET /image_codes/315f5939-0347-4f77-b494-fb4fba64eead/ HTTP/1.1" 200 3262
INFO 2024-04-28 09:49:28,365 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:49:33,876 basehttp 212 "GET /sms_codes/17752100458/?image_code=5evf&image_code_id=315f5939-0347-4f77-b494-fb4fba64eead HTTP/1.1" 200 69
INFO 2024-04-28 09:49:33,879 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:50:56,660 basehttp 212 "GET /sms_codes/17752100458/?image_code=5evf&image_code_id=315f5939-0347-4f77-b494-fb4fba64eead HTTP/1.1" 200 69
INFO 2024-04-28 09:50:56,662 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:57:28,829 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 09:57:31,331 basehttp 212 "GET /image_codes/aeea6a81-9fbd-4e01-a7e5-e1e41a7e47dd/ HTTP/1.1" 200 3247
INFO 2024-04-28 09:57:38,965 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 09:57:39,975 basehttp 212 "GET /sms_codes/17752100458/?image_code=F5UY&image_code_id=aeea6a81-9fbd-4e01-a7e5-e1e41a7e47dd HTTP/1.1" 200 27
INFO 2024-04-28 10:00:13,593 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 10:00:14,052 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 10:00:18,629 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 10:00:18,631 basehttp 212 "GET /sms_codes/17752100458/?image_code=F5UY&image_code_id=aeea6a81-9fbd-4e01-a7e5-e1e41a7e47dd HTTP/1.1" 200 69
INFO 2024-04-28 10:00:58,635 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 10:01:03,587 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 10:01:03,587 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
WARNING 2024-04-28 10:01:29,073 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 10:01:29,074 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
WARNING 2024-04-28 10:01:29,161 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 10:01:29,163 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
WARNING 2024-04-28 10:01:29,167 log 241 Not Found: /favicon.ico
WARNING 2024-04-28 10:01:29,168 basehttp 212 "GET /favicon.ico HTTP/1.1" 404 2994
WARNING 2024-04-28 10:01:32,618 log 241 Not Found: /register.html
WARNING 2024-04-28 10:01:32,619 basehttp 212 "GET /register.html HTTP/1.1" 404 3000
INFO 2024-04-28 10:01:45,340 basehttp 212 "GET /admin/ HTTP/1.1" 302 0
INFO 2024-04-28 10:01:45,370 basehttp 212 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4172
INFO 2024-04-28 10:01:45,403 basehttp 212 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2024-04-28 10:01:45,403 basehttp 212 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2024-04-28 10:01:45,403 basehttp 212 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2024-04-28 10:01:45,484 basehttp 212 "GET /static/admin/css/login.css HTTP/1.1" 304 0
INFO 2024-04-28 10:01:45,486 basehttp 212 "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2024-04-28 10:01:45,486 basehttp 212 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2024-04-28 10:01:45,495 basehttp 212 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
WARNING 2024-04-28 10:01:54,660 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 10:01:54,660 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 10:02:06,467 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 10:02:06,925 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 10:02:16,207 log 241 Not Found: /
WARNING 2024-04-28 10:02:16,208 basehttp 212 "GET / HTTP/1.1" 404 2943
WARNING 2024-04-28 10:02:39,479 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 10:02:39,479 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 10:02:52,826 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 10:02:53,300 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 10:03:29,774 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 10:03:36,040 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 10:03:36,040 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 15:52:24,004 autoreload 668 Watching for file changes with StatReloader
WARNING 2024-04-28 15:53:26,542 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 15:53:26,543 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 15:53:29,190 basehttp 212 "GET /image_codes/dfaa4ad6-1f96-4237-9f6b-aabaa211aa93/ HTTP/1.1" 200 3734
INFO 2024-04-28 15:53:35,348 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:53:47,958 basehttp 212 "GET /sms_codes/17369236240/?image_code=INMP&image_code_id=dfaa4ad6-1f96-4237-9f6b-aabaa211aa93 HTTP/1.1" 200 69
INFO 2024-04-28 15:53:47,960 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:55:27,052 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 15:55:32,496 basehttp 212 "GET /image_codes/004c0dd3-2789-4265-92f4-fd8a2664ad88/ HTTP/1.1" 200 3044
INFO 2024-04-28 15:55:35,174 basehttp 212 "GET /image_codes/57471d84-d0cb-4d7f-9305-f976f0dd3302/ HTTP/1.1" 200 3125
INFO 2024-04-28 15:55:44,196 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:55:51,184 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:55:56,657 basehttp 212 "GET /sms_codes/17369236240/?image_code=ltca&image_code_id=57471d84-d0cb-4d7f-9305-f976f0dd3302 HTTP/1.1" 200 27
INFO 2024-04-28 15:56:11,448 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 15:56:21,514 basehttp 212 "GET /image_codes/bff9c513-dd80-4d64-bc2f-7b4c1e821828/ HTTP/1.1" 200 3246
INFO 2024-04-28 15:56:27,473 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:56:34,102 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:58:20,762 basehttp 212 "GET /sms_codes/17369236240/?image_code=kfvh&image_code_id=bff9c513-dd80-4d64-bc2f-7b4c1e821828 HTTP/1.1" 200 27
INFO 2024-04-28 15:59:12,248 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 15:59:15,019 basehttp 212 "GET /image_codes/e140dbca-8803-4fc5-8343-8e8c91c46362/ HTTP/1.1" 200 2935
INFO 2024-04-28 15:59:23,126 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:59:27,707 basehttp 212 "GET /image_codes/54e19d39-e352-4fd5-871b-ca909845fe70/ HTTP/1.1" 200 2893
INFO 2024-04-28 15:59:33,866 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 15:59:43,499 basehttp 212 "GET /sms_codes/17369236240/?image_code=rqgk&image_code_id=54e19d39-e352-4fd5-871b-ca909845fe70 HTTP/1.1" 200 27
INFO 2024-04-28 15:59:49,032 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:02:18,387 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:02:27,563 basehttp 212 "GET /image_codes/5c84086b-3914-435e-8aa9-ed989f9b3c6c/ HTTP/1.1" 200 2972
INFO 2024-04-28 16:02:33,150 basehttp 212 "GET /usernames/yabnnn/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:02:45,651 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:02:48,074 basehttp 212 "GET /image_codes/40ced036-5384-4284-9b83-37a84aef4bd1/ HTTP/1.1" 200 2994
INFO 2024-04-28 16:02:57,352 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:06:01,733 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:06:02,975 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:06:10,186 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:06:11,483 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:06:19,408 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:06:40,019 basehttp 212 "GET /image_codes/e6b1356f-3918-4f24-a440-3cf1129314f7/ HTTP/1.1" 200 2910
INFO 2024-04-28 16:06:45,482 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:06:52,016 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:07:02,672 basehttp 212 "GET /sms_codes/17752100458/?image_code=8cne&image_code_id=e6b1356f-3918-4f24-a440-3cf1129314f7 HTTP/1.1" 200 27
INFO 2024-04-28 16:08:07,999 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:08:08,517 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:11:29,848 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:11:31,881 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:12:15,625 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:12:22,152 basehttp 212 "GET /image_codes/d5363551-040e-4709-9e70-aba8d2f231bb/ HTTP/1.1" 200 2919
INFO 2024-04-28 16:12:28,368 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:12:33,527 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:13:20,416 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:13:21,519 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:13:38,643 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:13:39,786 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:14:03,380 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:14:04,364 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:18:09,724 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:18:10,850 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:18:59,745 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:19:00,831 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:19:32,693 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:19:34,034 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:22:22,948 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:22:26,402 basehttp 212 "GET /image_codes/28c7de7a-eb6a-4f3f-bda6-7b1915919ee7/ HTTP/1.1" 200 3029
INFO 2024-04-28 16:22:33,494 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:22:42,776 basehttp 212 "GET /sms_codes/17752100458/?image_code=AF70&image_code_id=28c7de7a-eb6a-4f3f-bda6-7b1915919ee7 HTTP/1.1" 200 69
INFO 2024-04-28 16:22:42,781 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:23:38,871 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:23:44,211 basehttp 212 "GET /image_codes/a36399f6-5099-4c5a-8872-d8d1b3524a34/ HTTP/1.1" 200 3178
INFO 2024-04-28 16:23:50,768 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:23:55,146 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:25:03,147 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:25:03,510 basehttp 212 "GET /image_codes/884d0043-16f5-4b1f-80b9-9736448138c8/ HTTP/1.1" 200 3221
INFO 2024-04-28 16:25:08,491 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:25:14,779 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:26:03,786 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:26:04,924 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:26:08,095 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:26:09,345 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:26:15,098 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:26:22,456 basehttp 212 "GET /image_codes/84a15fef-8c2f-4ff7-b9d0-53449467e3bb/ HTTP/1.1" 200 3370
INFO 2024-04-28 16:26:28,708 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:26:33,555 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:30:19,228 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:30:20,433 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:30:20,925 basehttp 212 "GET /sms_codes/17752100458/?image_code=rsvk&image_code_id=84a15fef-8c2f-4ff7-b9d0-53449467e3bb HTTP/1.1" 200 69
INFO 2024-04-28 16:30:27,126 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:30:28,596 basehttp 212 "GET /image_codes/103f62b1-6418-4054-81d3-8724fca5bef6/ HTTP/1.1" 200 3304
INFO 2024-04-28 16:30:32,251 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:30:41,560 basehttp 212 "GET /sms_codes/17752100458/?image_code=mkld&image_code_id=103f62b1-6418-4054-81d3-8724fca5bef6 HTTP/1.1" 200 69
INFO 2024-04-28 16:30:41,571 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:31:21,100 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:31:23,912 basehttp 212 "GET /image_codes/6a004cce-49e5-47ff-a98e-2ed46e140bf1/ HTTP/1.1" 200 3097
INFO 2024-04-28 16:31:27,971 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:31:32,779 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:31:59,520 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:32:00,710 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:32:35,686 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:32:43,000 basehttp 212 "GET /image_codes/db3e8f2f-cc13-446c-87e4-dfece7e4c065/ HTTP/1.1" 200 3122
INFO 2024-04-28 16:32:47,142 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:32:52,248 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:32:59,307 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:34:05,577 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:34:06,825 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:34:11,841 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:34:12,407 basehttp 212 "GET /image_codes/8d5b11f6-47df-4f4c-aaef-4bea599b959c/ HTTP/1.1" 200 3338
INFO 2024-04-28 16:34:15,957 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:34:21,765 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:34:27,681 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:35:21,567 basehttp 212 "GET /sms_codes/17752100458/?image_code=olk4&image_code_id=8d5b11f6-47df-4f4c-aaef-4bea599b959c HTTP/1.1" 200 27
INFO 2024-04-28 16:36:16,659 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\libs\yuntongxun\sms.py changed, reloading.
INFO 2024-04-28 16:36:17,920 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:40:59,371 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:41:00,632 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:41:03,121 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:41:06,619 basehttp 212 "GET /image_codes/e058902f-9ce9-4e1b-8316-0beba167cd65/ HTTP/1.1" 200 2946
INFO 2024-04-28 16:41:10,083 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:41:17,719 basehttp 212 "GET /mobiles/17369236240/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:41:21,141 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:41:28,423 basehttp 81 - Broken pipe from ('127.0.0.1', 8275)
INFO 2024-04-28 16:41:40,100 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:41:48,942 basehttp 212 "GET /image_codes/4b04dc70-e669-4ac3-93dc-b6682a6eb0d0/ HTTP/1.1" 200 3106
INFO 2024-04-28 16:41:52,277 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:41:57,186 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:42:06,608 basehttp 212 "GET /sms_codes/17752100458/?image_code=hzeh&image_code_id=4b04dc70-e669-4ac3-93dc-b6682a6eb0d0 HTTP/1.1" 200 27
INFO 2024-04-28 16:42:29,505 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:42:30,094 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:42:34,606 basehttp 212 "GET /image_codes/f887966b-0585-4855-b719-d2e089d2819f/ HTTP/1.1" 200 3186
INFO 2024-04-28 16:42:37,588 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:42:39,196 basehttp 212 "GET /image_codes/d9294b73-6dbc-4e2c-aca9-6f53c8d3960b/ HTTP/1.1" 200 3382
INFO 2024-04-28 16:42:43,652 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:42:53,314 basehttp 212 "GET /sms_codes/17752100458/?image_code=n5lq&image_code_id=d9294b73-6dbc-4e2c-aca9-6f53c8d3960b HTTP/1.1" 200 27
INFO 2024-04-28 16:43:04,061 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 16:43:04,623 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:43:07,610 basehttp 212 "GET /image_codes/f02c2a52-b110-4acc-a10c-c93f514dbe97/ HTTP/1.1" 200 3511
INFO 2024-04-28 16:43:10,410 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:43:17,810 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 16:43:27,324 basehttp 212 "GET /sms_codes/17752100458/?image_code=ny6g&image_code_id=f02c2a52-b110-4acc-a10c-c93f514dbe97 HTTP/1.1" 200 27
INFO 2024-04-28 16:48:44,380 basehttp 212 "GET /image_codes/f117562e-c82f-48d0-9051-fe5b9757cfcc/ HTTP/1.1" 200 3156
INFO 2024-04-28 16:53:41,213 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:53:41,612 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:54:17,712 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:54:18,174 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:54:59,172 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:54:59,578 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:55:44,909 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:55:45,349 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:55:51,735 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:55:52,171 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:56:01,174 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:56:01,645 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:56:07,025 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:56:07,475 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:56:25,167 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:56:25,576 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:56:34,422 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:56:34,870 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 16:57:20,336 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 16:57:20,822 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:01:55,761 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 17:01:56,174 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:02:22,670 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 17:02:23,149 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:02:43,441 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\users\views.py changed, reloading.
INFO 2024-04-28 17:02:43,883 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:02:48,217 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:02:53,152 basehttp 212 "GET /image_codes/6ab56c97-da27-4e35-91b0-b9214b4ff129/ HTTP/1.1" 200 3348
INFO 2024-04-28 17:02:58,137 basehttp 212 "GET /usernames/yanming/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:03:07,981 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:03:12,553 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:03:22,980 basehttp 212 "GET /sms_codes/17752100458/?image_code=eigx&image_code_id=6ab56c97-da27-4e35-91b0-b9214b4ff129 HTTP/1.1" 200 27
INFO 2024-04-28 17:03:36,724 basehttp 212 "OPTIONS /register/ HTTP/1.1" 200 0
INFO 2024-04-28 17:03:36,725 basehttp 212 "GET /usernames/yanming/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:03:36,726 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:03:36,988 basehttp 212 "POST /register/ HTTP/1.1" 200 27
WARNING 2024-04-28 17:03:41,778 log 241 Not Found: /carts/simple/
WARNING 2024-04-28 17:03:41,779 basehttp 212 "GET /carts/simple/ HTTP/1.1" 404 3000
INFO 2024-04-28 17:08:31,748 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:08:32,184 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:09:00,397 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:09:00,901 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:09:30,717 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:09:31,098 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:09:36,797 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:09:37,312 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:09:43,134 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:09:43,638 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:09:51,175 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:09:51,565 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:10:05,298 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:10:05,787 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:10:34,571 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:10:35,089 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:10:38,818 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:10:39,308 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:10:57,245 basehttp 212 "GET /image_codes/477b72bf-fb72-47c8-8f14-5f19019571e8/ HTTP/1.1" 200 3195
INFO 2024-04-28 17:11:01,348 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:11:01,349 basehttp 212 "GET /usernames/yanming/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:11:03,807 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:11:14,934 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:11:18,310 basehttp 212 "GET /sms_codes/17752100458/?image_code=eigx&image_code_id=477b72bf-fb72-47c8-8f14-5f19019571e8 HTTP/1.1" 200 69
INFO 2024-04-28 17:11:18,311 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:23:31,237 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:23:31,738 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:23:54,259 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:23:54,750 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:24:01,591 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:24:02,091 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:24:11,017 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:24:11,414 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:25:04,919 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:25:05,402 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:25:58,171 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:25:58,676 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:26:37,441 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:26:45,019 basehttp 212 "GET /image_codes/97372e17-94fc-48b6-91ae-6080ed4c8f82/ HTTP/1.1" 200 3432
INFO 2024-04-28 17:26:51,660 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:26:58,951 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:30:08,315 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:30:12,963 basehttp 212 "GET /image_codes/39146ab9-fd39-4aae-b93b-5b4b4ca37b6e/ HTTP/1.1" 200 2813
INFO 2024-04-28 17:30:15,471 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
INFO 2024-04-28 17:30:24,193 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 17:30:38,999 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 35, in get
    send_flag=redis_cil.sadd('send_flag_%s'%mobile)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 3315, in sadd
    return self.execute_command("SADD", name, *values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 522, in _send_command_parse_response
    return self.parse_response(conn, command_name, **options)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 562, in parse_response
    response = connection.read_response()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 536, in read_response
    raise response
redis.exceptions.ResponseError: wrong number of arguments for 'sadd' command
ERROR 2024-04-28 17:30:47,959 basehttp 212 "GET /sms_codes/17752100458/?image_code=h4ti&image_code_id=39146ab9-fd39-4aae-b93b-5b4b4ca37b6e HTTP/1.1" 500 130722
INFO 2024-04-28 17:31:54,206 autoreload 668 Watching for file changes with StatReloader
INFO 2024-04-28 17:31:59,287 basehttp 212 "GET /image_codes/13877d7d-a1b0-4109-948a-7873e1fa96bb/ HTTP/1.1" 200 3238
INFO 2024-04-28 17:32:04,383 basehttp 212 "GET /mobiles/17752100458/count/ HTTP/1.1" 200 39
ERROR 2024-04-28 17:32:04,410 log 241 Internal Server Error: /sms_codes/17752100458/
Traceback (most recent call last):
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py", line 35, in get
    send_flag=redis_cil.sadd('send_flag_%s'%mobile)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\commands\core.py", line 3315, in sadd
    return self.execute_command("SADD", name, *values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 545, in execute_command
    return conn.retry.call_with_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\retry.py", line 46, in call_with_retry
    return do()
           ^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 546, in <lambda>
    lambda: self._send_command_parse_response(
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 522, in _send_command_parse_response
    return self.parse_response(conn, command_name, **options)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\client.py", line 562, in parse_response
    response = connection.read_response()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\桌面\计算机\全栈\django\ven_dj\Lib\site-packages\redis\connection.py", line 536, in read_response
    raise response
redis.exceptions.ResponseError: wrong number of arguments for 'sadd' command
ERROR 2024-04-28 17:32:04,413 basehttp 212 "GET /sms_codes/17752100458/?image_code=dhct&image_code_id=13877d7d-a1b0-4109-948a-7873e1fa96bb HTTP/1.1" 500 116790
INFO 2024-04-28 17:32:48,183 autoreload 266 E:\桌面\计算机\全栈\django\django项目\tuling\apps\verification\views.py changed, reloading.
INFO 2024-04-28 17:32:48,616 autoreload 668 Watching for file changes with StatReloader
